### PR TITLE
Added Natwest UK to list

### PR DESCRIPTION
**Pull request by eddiejaoude**
No `tfa` :(

### CODE REVIEW

**Comment by Carlgo11** [comment]
You need to add a logo. 32x32px png

**Comment by eddiejaoude** [comment]
Oh I see, I thought it was the `uri` to the image. I will sort it.

**Comment by Carlgo11** [comment]
It's url not uri. https://en.wikipedia.org/wiki/Uniform_resource_locator

**Comment by Carlgo11** [comment]
Hmm the logo seems very low quality. How about this one instead?
natwest

File: _data/banking.yml [hunk 1]
```diff
@@ -144,6 +144,12 @@ websites:
       img: lloydsbank.png
       tfa: No
 
+    - name: Natwest UK
+      url: http://www.natwest.com/personal.ashx
```
Comment (Carlgo11) [inline _data/banking.yml]: The site supports https. Use that instead

Comment (eddiejaoude) [inline _data/banking.yml]: Sure.

**Comment by dsoegijono** [comment]
:+1:

**Pull request merged**